kube/service.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: little-help-book
+
+  namespace: little-help-book
+  annotations:
+    getambassador.io/config: |
+      ---
+      apiVersion: ambassador/v0
+      kind: Mapping
+      name: stage-little-help-book-mapping
+      host: mvpstudio.stage.apps.mvpstudio.org
+      prefix: /api
+      rewrite: /api
+      service: mvpstudio-web-api.stage-mvpstudio-web
```

Comment:
This is the wrong service. You actually need to create a service for this first and then use that here.

Reply:
I tried to fix this in [PR#3](https://github.com/OpenEugene/little-help-book/pull/3).